components/checkbox/README.md

```diff
@@ -8,66 +8,67 @@ author: orinevares
 ![Status](https://img.shields.io/badge/Component-Draft-orange.svg)
 
 # Checkbox
-Checkboxes are a form of control inputs that allow users to select one or more options from a list.
+Checkboxes are a form of control inputs that allow users to select one or more items from a list.
 
 ## Required
 
-## Last Updated: 2 days ago
+## Last Updated:
 
 ## Visual
-This is where the visual goes
+Checkbox Visual:
+![Checkbox](https://github.com/bcgov/design-system/blob/master/components/checkbox/images/checkbox.png?raw=true)
 
-## How to Use This:
-When users have to:
-* Select multiple options from a given list
-* Toggle a single option on or off
+## Requirements
+* This component must be used according the the standard outined in this document. Any feedback should be directed to the feedback forum.
 
-Tips:
+## Use This For:
+* Selecting multiple options from a given list
+* Toggling a single option on or off
+
+## Don't Use This when:
+* Users can only select one option from a list, use a radio button instead.
+
+## Design Protocol:
 * List options vertically; horizontal listings can make it difficult to tell which label pertains to which checkbox.
 * Avoid using negative language in labels. Use “I want to receive email updates” instead of “I don’t want to receive email updates.”
 * Make sure selections are adequately spaced for touch screens.
 * Do not assume users will know how many options they can select. If needed, add a hint such as “Select all that apply” 
 
-## Don't Use This for:
-* If a user can only select one option from a list, use a radio button instead.
-
 ## Rationale
 Our checkboxes are larger than most browser defaults to allow for greater visibility and larger target areas for touchscreen devices. Research has shown that users resort to only clicking the checkbox and not the associated text, therefore a larger target area is necessary. Simply enlarging browser defaults proved to be problematic on many web browsers and the custom design should be used.
-Based on research done by Gov.UK: https://designnotes.blog.gov.uk/2016/11/30/weve-updated-the-radios-and-checkboxes-on-gov-uk/
+Based on [research](https://designnotes.blog.gov.uk/2016/11/30/weve-updated-the-radios-and-checkboxes-on-gov-uk/) done by Gov.UK.
 
 ## Behaviour
 1. Users should be able to tap or click on either the checkbox or the label to select the option.
+2. Do not preselect options, this signals a biased result.
```

Comment:
Comma splice - change to: "Do not preselect options. This signals a biased result."